### ISSUE TITLE
Question: POST body with "params" key

### ISSUE THREAD

**Issue by StoyanKostov** [smollweide/node-mock-server#74]
Hello,

I have a POST request with a body like:
```
{
  "method": "DEL",
  "params": {
    "application": "LOANS",
    "record_id": "LD111111111111"
  }
}
```
My response should be something like:
```
{
    "record_id": "<%-getId(params);%>",
    "operation_successful": true
}
```
My getId function is:
```
....
getId: function ( params ) {
        return params.application;
    }
....
```
It seems to me that "params" is a reserved word for  GET url parameters. So "params" has a value of empty object {} in getId function.  How can I access "application" value in getId?

Best regards!

**Comment by smollweide** [comment]
Hello @StoyanKostov,
thanks for this report.
You are right "params" is reserved for path params.
Could be possible to provide the data in a wrapped object like:

```
{
  "reponseBody": {},
  "params": {},
  "query": {},
  "faker": {},
  ...
}
```

Best

**Comment by smollweide** [comment]
Fix was published as 0.19.0-alpha.
Run `npm install node-mock-server@0.19.0-alpha`
and change your getId function:
```
....
getId: function ( params ) {
        return body.params.application;
    }
....
```
